esmvaltool/utils/recipe_test_workflow/site/dkrz.cylc

```diff
@@ -0,0 +1,57 @@
+#!jinja2
+[runtime]
+    [[root]]
+        [[[environment]]]
+            CONTAINER_DIR = ${ROSE_DATAC}/container
+            CONTAINER_FILE = esmvaltool.sif
+            CONTAINER_PATH = ${CONTAINER_DIR}/${CONTAINER_FILE}
+            # Warning: fragile assumption of location of esmvaltool code,
+            # issue #3437 contains more details.
+            ESMVALTOOL_DIR = /opt/conda/envs/esmvaltool/lib/python3.12/site-packages
+
+    # COMPUTE provides defaults for computation-heavy tasks.
+    # Specific tasks below override some defaults, e.g. time & memory.
+    [[COMPUTE]]
+        platform = levante
+        execution time limit = PT3M
+        [[[directives]]]
+            --wckey = RTW
+            --account = bk1088
+            --partition = interactive
+            --mem = 15G
+
+    [[install_env_file]]
+        [[[environment]]]
+            ROSE_APP_OPT_CONF_KEYS = {{ SITE }}
+            SINGULARITY_ENV_FILE=singularity-env
+            SINGULARITY_ENV_FILE_SITE_PATH=${CYLC_WORKFLOW_RUN_DIR}/site/${SITE}-singularity-env
+
+    [[get_esmval]]
+        platform = localhost
+        execution time limit = PT10M  # Actual: 2m17s on 2024-12-06.
+        [[[environment]]]
+            SINGULARITY_CACHEDIR = ${CONTAINER_PATH}
```

Comment:
Please move to `dkrz-env`, with a comment to explain what the var does

Reply:
Following some offline discussions, I have moved `SINGULARITY_CACHEDIR` to the `dkrz-singularity-env` environment file rather than the `dkrz-env` environment file: 8ec835e.

The reason for this is that `SINGULARITY_CACHEDIR` was originally added to the environment for the `get_esmval` app (which uses the `dkrz-singularity-env` environment file directly rather than the `dkrz-env` environment file).

Also, if I'm reading the [singularity documentation](https://docs.sylabs.io/guides/latest/user-guide/build_env.html) correctly, `SINGULARITY_CACHEDIR` is only used when builing containers, but on the off chance it is used by other singularity commands, this solution should cover that, as `dkrz-env` uses `dkrz-singularity-env` 💪